jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/surfaces/AccordionTests.java

```diff
@@ -1,35 +1,47 @@
 package io.github.epam.material.tests.surfaces;
 
 import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.surfaces.AccordionPage.*;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
 
-import static io.github.com.StaticSite.accordionFrame;
-import static io.github.com.StaticSite.surfaceAccordionDisabledPage;
-import static io.github.com.StaticSite.surfaceAccordionPage;
+/**
+ * To see an example of Accordion web element please visit
+ * https://jdi-testing.github.io/jdi-light/material/accordion_surface
```

Comment:
Please use link https://material-ui.com/components/accordion/#accordion
The same for file Accordion.java